openprescribing/frontend/management/commands/create_views.py

```diff
@@ -88,17 +90,39 @@ def fill_views(self):
         pool.join()  # wait for all worker processes to exit
 
         for table in tables:
-            self.download_and_import(table)
+            self.download_and_import(table, prescribing_date)
             self.log("-------------")
 
-    def download_and_import(self, table):
+    def download_and_import(self, table, prescribing_date):
         storage_prefix = 'hscic/views/{}-'.format(table.name)
         exporter = TableExporter(table, storage_prefix)
 
-        with tempfile.NamedTemporaryFile(mode='r+') as f:
+        filename_base = '{}-{}'.format(table.name, prescribing_date)
```

Comment:
Simply using `tempfile` should work, because we set `export TMPDIR=/mnt/database/tmp/` in the production environment.

As the location of large volumes is environment-specific, I think that's the right way of doing this?

(Though ISTR there was an issue with a disk filling up when you were trying this out before...?)